camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -664,6 +672,12 @@ export default class Tasklist extends Mixins(TaskListMixin) {
     });
     await viewer.importXML(this.xmlData);
     viewer.get("canvas").zoom("fit-viewport");
+    for(let i: number=0; i<activityList.length; i++){
```

Comment:
Use const instead of let, as no modification required

Reply:
we can't assign the variable in const inside the for loop. const is constant we can't change . In for loop we need to iterate so we need to increment the value. that's why I am using let here

Reply:
You are correct missed it :)